packages/react-components/src/trace-explorer/trace-explorer-time-range-data-widget.tsx

```diff
@@ -267,6 +267,19 @@ export class ReactTimeRangeDataWidget extends React.Component<
         }
     };
 
+    /**
+     * Allows public to set active data and experiment data map.
+     * This is used in vscode-trace-extension where we need to manually
```

Comment:
we should not mention vscode in the comment since vscode is just on implementation.

Reply:
Ok.